stock_account_internal_move/models/stock_move.py

```diff
@@ -0,0 +1,66 @@
+# Copyright (C) 2018 by Camptocamp
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo import api, models
+from odoo.tools import float_round
+
+
+class StockMove(models.Model):
+    _inherit = 'stock.move'
+
+    @api.multi
+    def _action_done(self):
+        res = super()._action_done()
+
+        for move in res:
+            # first of all, define if we need to even valuate something
+            if move.product_id.valuation != 'real_time':
+                continue
```

Comment:
@arkostyuk I don't understand this check. The accounting entry needs to be created for every costing method.

Reply:
@arkostyuk You're right, sorry. I haven't had my coffee. I mixed real time and real price...